specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimapis.json

```diff
@@ -0,0 +1,1878 @@
+            "required": true,
+            "description": "The entity state (Etag) version of the Api Operation Policy to delete. A value of \"*\" can be used for If-Match to unconditionally apply the operation.",
+            "type": "string"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/PolicyIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Successfully deleted the policy configuration at the API Operation level."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/apis/{apiId}/products": {
+      "get": {
+        "tags": [
+          "ApiProduct"
+        ],
+        "operationId": "ApiProduct_ListByApis",
+        "description": "Lists all Products, which the API is part of.",
+        "x-ms-examples": {
+          "ApiManagementListApisProducts": {
+            "$ref": "./examples/ApiManagementListApisProducts.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "| Field | Supported operators    | Supported functions                         |\n|-------|------------------------|---------------------------------------------|\n| name  | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/TopQueryParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SkipQueryParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation returns a collection of products which have the Api entity.",
+            "schema": {
+              "$ref": "./apimproducts.json#/definitions/ProductCollection"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-odata": "./apimproducts.json#/definitions/ProductContract"
+      }
+    },
+    "/apis/{apiId}/policies": {
+      "get": {
+        "tags": [
+          "ApiPolicy"
+        ],
+        "operationId": "ApiPolicy_ListByApi",
```

Comment:
>ApiPolicy_ListByApi [](start = 24, length = 19)

This could be pageable

Reply:
This is only one policy.

---
In reply to: [163345553](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163345553) [](ancestors = 163345553)

Reply:
Isn't it a list? :(

Reply:
It is with only one policy.

---
In reply to: [163348702](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163348702) [](ancestors = 163348702)

Reply:
I see, could wording be changed? This can be confusing for users.